### PR TITLE
Fixed #7674: Deletes file from assets folder, not just the database reference.

### LINKED ISSUE #7674: Files attached to assets to actually be deleted
**Server (please complete the following information):**
 - Snipe-IT Version 4.8.0
 - OS: Debian 10
 - Web Server: Apache

**Is your feature request related to a problem? Please describe.**
Files attached to assets (and other things too, I'm assuming) via the file tab don't actually get deleted from the server after pressing the "delete" button and can be found in /storage/private_uploads/assets. They are also included in backups, which leads to significant bloat and potentially confidential documents sitting there, forever. The purge option doesn't include them as well.
Unless I'm missing something, of course, and there's something wrong on my end.

**Describe the solution you'd like**
An option for the files to be physically deleted would be very welcome, since manually removing them every time isn't a solution in the long run.

### PR DESCRIPTION
**Pull request by misilot**
# Description

The function ```destroy``` in the AssetFilesController.php only removes the link in the database, but does not clean up the actual file
Fixes #7674 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Upload File under the Asset -> Files tab
- [ ] Try Deleting file
- [ ] Check private_uploads/assets/ -- file will still be there.
- [ ] Remove file
- [ ] Apply patch
- [ ] Upload file under assets -> files tab
- [ ] Try deleting file
- [ ] Check private_uploads/assets/ -- file will be removed

**Test Configuration**:
* PHP version: 7.2.24
* MySQL version: MySQL 5.7
* Webserver version - Apache 2.4.29
* OS version - ubuntu 18.04

# Checklist:

- [X] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [X] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/Http/Controllers/Assets/AssetFilesController.php [hunk 1]
```diff
@@ -110,15 +110,15 @@ public function destroy($assetId = null, $fileId = null)
     {
         $asset = Asset::find($assetId);
         $this->authorize('update', $asset);
-        $rel_path = 'storage/private_uploads/assets';
```
Comment (snipe) [inline app/Http/Controllers/Assets/AssetFilesController.php:113]: Actually, why did you remove the `storage` from the path here? `private_uploads` lives in the `storage` directory.

Comment (misilot) [reply app/Http/Controllers/Assets/AssetFilesController.php:113]: Because looking higher up in the file it appears the Storage service is relative to that storage directory (See: https://github.com/misilot/snipe-it/blob/89de8c2213cfad83a40b3959ace9f9563d654979/app/Http/Controllers/Assets/AssetFilesController.php#L41).

I think that is partially why it wasn't being removed correctly.

File: app/Http/Controllers/Assets/AssetFilesController.php [hunk 2]
```diff
@@ -110,15 +110,15 @@ public function destroy($assetId = null, $fileId = null)
...
+        $rel_path = 'private_uploads/assets';
 
         // the asset is valid
         if (isset($asset->id)) {
             $this->authorize('update', $asset);
             $log = Actionlog::find($fileId);
             if ($log) {
-            if (file_exists(base_path().'/'.$rel_path.'/'.$log->filename)) {
-                Storage::disk('public')->delete($rel_path.'/'.$log->filename);
```
Comment (misilot) [inline app/Http/Controllers/Assets/AssetFilesController.php:121]: The asset is stored in the private store, and not public.
